crates/sui-graphql-rpc/src/client/simple_client.rs

```diff
@@ -18,6 +21,50 @@ use crate::{
 };
 
 use super::response::GraphqlResponse;
+
+pub struct GraphqlVariable {
```

Comment:
nit: maybe a better name is QueryVariable?

Reply:
Sure, renamed.